odk1-src/form-logic.rst

```diff
@@ -225,6 +228,22 @@ put the response value in the :th:`default` column.
   contacts, text_message, Text message
   contacts, email, Email
   
+.. tip:: 
+
+  You may want to use a previously entered value as a default,
+  but the :th:`default` column does not accept dynamic values.
+  
+  To workaround this, use the :th:`calculation` column instead,
+  and wrap your default value expression in a :func:`once` function.
+  
+  .. rubric:: XLSForm
+  
+  .. csv-table:: survey
+    :header: type, name, label, calculation
+    
+    text, default_value, Enter any text.,
+    text, calculated_default, Leave the value or change it., once(${default_value})
```

Comment:
This works if you want the default value to be exactly the FIRST value that ${default_value} is ever set to.

I would highlight two limitations:
- If the value of ${default_value} changes *even before the user gets to the ${calculated_default} question*, the default will match the original value of ${default_value}, not the updated one.
- If ${default_value} has a value, ${calculated_default} MUST also have a value. If the user sets it to blank, it will go back to matching ${default_value} (which may or may not have changed, as described in the first point).

The wording needs some work but hopefully the concepts make sense.

Reply:
Perhaps we can have an example?

age
gender
malaria_age
favorite_color

If I set age to 8, and go to malaria_age, it will be set to 8.

If I set age to 8, go to gender, go back to age and set it to 12, and go to malaria_age, it will be 8. This is because the expression in once() was evaluated when I first set age.

If I set age to 8, go to gender, go back to age and set it to 12, and go to malaria_age and set it blank, and go to favorite_color, malaria_age will be set to 12. This is because the expression in once() was re-evaluated when I set malaria_age to blank and the value of age was pulled at that time.